python/tvm/relay/transform.py

```diff
@@ -57,10 +59,99 @@ class PassContext(RelayNode):
     Each pass context contains a number of auxiliary information that is used
     to help an optimization pass. Such information includes the error reporter
     to record the errors of during the optimization, etc.
+
+    opt_level : Optional[int]
+        The optimization level of this pass.
+
+    fallback_device : Optional[int]
+        The fallback device type. It is also used as the default device for
+        operators that are not annotated during heterogeneous execution.
+
+    required_pass : Optional[List[str]]
+        The list of passes that are required by a certain pass.
+
+    disabled_pass : Optional[List[str]]
+        The list of passes that are disabled.
     """
+    defaults = {
+        "opt_level": 2,
+        "required_pass": None,
+        "disabled_pass": None,
+        "fallback_device": _nd.cpu(),
+    }
+
+    def __init__(self, **kwargs):
```

Comment:
consider not to use kwargs, instead directly list all the arguments and defaults